test/specs/sdk.specs.ts

```diff
@@ -50,4 +50,25 @@ describe("SDK", () => {
                 .equal(sdk.api);
         }
     });
+
+    it("should add listener on the sdk", () => {
+        const sdk = new SDK({ endpoint: "/" });
+
+        const expectedPromise = new Promise((resolve) => sdk.addListener("test", () => resolve(true)));
+
+        sdk.emit("test");
+
+        expect(expectedPromise).to.eventually.be.fulfilled;
```

Comment:
The test is pretty straight-forward. We do not use the p-event library otherwise so I do not know if we need to introduce this dependency for only that code. Thoughts?

Reply:
yeah no need!